lib/kennel.rb

```diff
@@ -54,48 +54,88 @@ class << self
   self.err = $stderr
 
   class Engine
-    attr_accessor :strict_imports
+    def initialize(
+      update_generated: true,
+      show_plan: false,
+      require_confirm: nil,
```

Comment:
```suggestion
      require_confirm: default_require_confirm?,
```

Reply:
It changes the behaviour of `.new(require_confirm: nil, ...)`, but I suppose that's OK :-) It wasn't defined either way.